nodebuilder/node_test.go

```diff
@@ -33,7 +33,8 @@ func TestLifecycle(t *testing.T) {
 		t.Run(strconv.Itoa(i), func(t *testing.T) {
 			// we're also creating a test node because the gRPC connection
 			// is started automatically when starting the node.
-			host, port, err := net.SplitHostPort(core.StartTestNode(t).GRPCClient.Target())
+			consNode := core.StartTestNode(t)
+			host, port, err := net.SplitHostPort(consNode.GRPCClient.Target())
 			require.NoError(t, err)
```

Comment:
combined with the below TestnodeWithConfig call, we were spinning up three different consensus networks afaiu

maybe I'm missing something simple